### PR TITLE
Fix: _WD_DebugSwitch stack update

### PR DESCRIPTION
**Pull request by Danp2**
### Proposed changes

Correctly store updated debug level on the stack

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [x] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

Prior stack value is overritten

### What is the new behavior?

New value is correctly add at the end of the stack

### System under test

Please complete the following information.

- OS: [e.g. Windows 10]
- OS Arch.: [e.g. X64]
- Browser [e.g. firefox]
- Browser version [e.g. 96.0.3]

### CODE REVIEW

**Comment by Danp2** [comment]
@mlipok Please double check this minor correction

File: wd_helper.au3 [hunk 1]
```diff
@@ -477,7 +477,7 @@ Func _WD_DebugSwitch($vMode = Default, $iErr = @error, $iExt = @extended)
 		$iResult = 1
 	ElseIf IsInt($vMode) And $vMode >= $_WD_DEBUG_None And $vMode <= $_WD_DEBUG_Full Then ; setting new debug level
 		ReDim $a_WD_DEBUG_SavedStack[$iStackSize + 1] ; resize / add new position to the stack
-		$a_WD_DEBUG_SavedStack[$iStackSize - 1] = $vMode ; set new last stack value
+		$a_WD_DEBUG_SavedStack[$iStackSize] = $vMode ; set new last stack value
```
Comment (mlipok) [inline wd_helper.au3:480]: oh. I see my mistake.
After **ReDim** last element ID should not be decrased.
But will check it deeply in a moment.

**Comment by mlipok** [comment]
Take a look at my recent commit

and check it with :
```autoit
Func UserTesting() ; here you can replace the code to test your stuff before you ask on the forum

	ConsoleWrite(@ScriptLineNumber & ' START: ' & $_WD_DEBUG & @CRLF)

	_WD_DebugSwitch($_WD_DEBUG_None)
	ConsoleWrite(@ScriptLineNumber & ' After $_WD_DEBUG_None: ' & $_WD_DEBUG & @CRLF)
	_WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' Restoring: ' & $_WD_DEBUG & @CRLF)

	_WD_DebugSwitch($_WD_DEBUG_Error)
	ConsoleWrite(@ScriptLineNumber & ' After $_WD_DEBUG_Error: ' & $_WD_DEBUG & @CRLF)
	_WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' Restoring: ' & $_WD_DEBUG & @CRLF)

	_WD_DebugSwitch($_WD_DEBUG_Info)
	ConsoleWrite(@ScriptLineNumber & ' After $_WD_DEBUG_Info: ' & $_WD_DEBUG & @CRLF)
	_WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' Restoring: ' & $_WD_DEBUG & @CRLF)

	_WD_DebugSwitch($_WD_DEBUG_Full)
	ConsoleWrite(@ScriptLineNumber & ' After $_WD_DEBUG_Full: ' & $_WD_DEBUG & @CRLF)
	_WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' Restoring: ' & $_WD_DEBUG & @CRLF)

EndFunc   ;==>UserTesting
```

my results are:
```
+ wd_demo.au3: Running: UserTesting
1088 START: 2
1091 After $_WD_DEBUG_None: 0
_WD_DebugSwitch ==> Success [0 / 0]
1093 Restoring: 2
1096 After $_WD_DEBUG_Error: 1
_WD_DebugSwitch ==> Success [0 / 0]
1098 Restoring: 2
_WD_DebugSwitch ==> Success [0 / 0]
1101 After $_WD_DEBUG_Info: 2
_WD_DebugSwitch ==> Success [0 / 0]
1103 Restoring: 2
_WD_DebugSwitch ==> Success [0 / 0]
1106 After $_WD_DEBUG_Full: 3
_WD_DebugSwitch ==> Success [0 / 0]
1108 Restoring: 2
+ wd_demo.au3: Finished: UserTesting

```

**Comment by mlipok** [comment]
also have one new idea...

**Comment by mlipok** [comment]
take a look also on:
[Success - current stack size](https://github.com/Danp2/au3WebDriver/pull/423/commits/b38a7ce385839b1869bb4cea99dc2e98981ef898)

and test it with:
```autoit
Func UserTesting() ; here you can replace the code to test your stuff before you ask on the forum

	ConsoleWrite(@ScriptLineNumber & ' START: 					$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)
	Local $iResult

	ConsoleWrite("! " & @CRLF)
	$iResult = _WD_DebugSwitch($_WD_DEBUG_None)
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     After $_WD_DEBUG_None:	$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)
	$iResult = _WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     Restoring:			$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)

	ConsoleWrite("! " & @CRLF)
	$iResult = _WD_DebugSwitch($_WD_DEBUG_Error)
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     After $_WD_DEBUG_Error:	$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)
	$iResult = _WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     Restoring:			$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)

	ConsoleWrite("! " & @CRLF)
	$iResult = _WD_DebugSwitch($_WD_DEBUG_Info)
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     After $_WD_DEBUG_Info:	$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)
	$iResult = _WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     Restoring:			$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)

	ConsoleWrite("! " & @CRLF)
	$iResult = _WD_DebugSwitch($_WD_DEBUG_Full)
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     After $_WD_DEBUG_Full:	$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)
	$iResult = _WD_DebugSwitch()
	ConsoleWrite(@ScriptLineNumber & ' RESULT=' & $iResult & '     Restoring:			$_WD_DEBUG=' & $_WD_DEBUG & @CRLF)

EndFunc   ;==>UserTesting
```

my resultas are:
```
+ wd_demo.au3: Running: UserTesting
1088 START: 					$_WD_DEBUG=2
! 
1093 RESULT=2     After $_WD_DEBUG_None:	$_WD_DEBUG=0
_WD_DebugSwitch ==> Success [0 / 0]
1095 RESULT=1     Restoring:			$_WD_DEBUG=2
! 
1099 RESULT=2     After $_WD_DEBUG_Error:	$_WD_DEBUG=1
_WD_DebugSwitch ==> Success [0 / 0]
1101 RESULT=1     Restoring:			$_WD_DEBUG=2
! 
_WD_DebugSwitch ==> Success [0 / 0]
1105 RESULT=2     After $_WD_DEBUG_Info:	$_WD_DEBUG=2
_WD_DebugSwitch ==> Success [0 / 0]
1107 RESULT=1     Restoring:			$_WD_DEBUG=2
! 
_WD_DebugSwitch ==> Success [0 / 0]
1111 RESULT=2     After $_WD_DEBUG_Full:	$_WD_DEBUG=3
_WD_DebugSwitch ==> Success [0 / 0]
1113 RESULT=1     Restoring:			$_WD_DEBUG=2
+ wd_demo.au3: Finished: UserTesting
```

File: wd_helper.au3 (hunk 1 above)
Comment (Danp2) [reply wd_helper.au3]: Please see my last commit, which fixes another issue with the above. Here's a modified version of your code that I used for testing --

```autoit
#include "..\webdriver\wd_helper.au3"

ConsoleWrite(' START: ' & $_WD_DEBUG & @CRLF)

_WD_DebugSwitch($_WD_DEBUG_None)
ConsoleWrite(' After $_WD_DEBUG_None: ' & $_WD_DEBUG & @CRLF)
_WD_DebugSwitch()
ConsoleWrite(' Restoring: ' & $_WD_DEBUG & @CRLF)

_WD_DebugSwitch($_WD_DEBUG_Error)
ConsoleWrite(' After $_WD_DEBUG_Error: ' & $_WD_DEBUG & @CRLF)

_WD_DebugSwitch($_WD_DEBUG_Info)
ConsoleWrite(' After $_WD_DEBUG_Info: ' & $_WD_DEBUG & @CRLF)
_WD_DebugSwitch()
ConsoleWrite(' Restoring: ' & $_WD_DEBUG & @CRLF)

_WD_DebugSwitch()
ConsoleWrite(' Restoring: ' & $_WD_DEBUG & @CRLF)

_WD_DebugSwitch($_WD_DEBUG_Full)
ConsoleWrite(' After $_WD_DEBUG_Full: ' & $_WD_DEBUG & @CRLF)
_WD_DebugSwitch()
ConsoleWrite(' Restoring: ' & $_WD_DEBUG & @CRLF)
```

and my results --

```
     START: 2
     After $_WD_DEBUG_None: 0
    _WD_DebugSwitch ==> Success [0 / 0]
     Restoring: 2
     After $_WD_DEBUG_Error: 1
    _WD_DebugSwitch ==> Success [0 / 0]
     After $_WD_DEBUG_Info: 2
     Restoring: 1
    _WD_DebugSwitch ==> Success [0 / 0]
     Restoring: 2
    _WD_DebugSwitch ==> Success [0 / 0]
     After $_WD_DEBUG_Full: 3
    _WD_DebugSwitch ==> Success [0 / 0]
     Restoring: 2
```

**Pull request closed without merging**